### PR TITLE
🚚Screenshot service to Mercury

### PR DESCRIPTION
**Pull request by FaazAbidi**

### CODE REVIEW

File: tests/test_all.py [hunk 1]
```diff
@@ -111,7 +104,7 @@ def test_getrich_spotify_album():
     requested = requests.get(URL).text
     get_data = main_generic(requested, URL)
     enriched_data = enrich_test(URL)
-    assert enriched_data == test_enrich_spotify_album_output
+    # assert enriched_data == test_enrich_spotify_album_output
```
Comment (FaazAbidi) [inline tests/test_all.py:107]: @alihaiderrizvi asked me to comment this out. This test case has some issues. He'll fix it in the next PR.

**Comment by FaazAbidi** [comment]
@joelrojo I'm done!

File: tests/test_all.py (hunk 1 above)
Comment (joelrojo) [reply tests/test_all.py:107]: Cool no worries!

**Review by joelrojo — APPROVED** [review]
Thanks @FaazAbidi looks awesome!

**Pull request merged**